reverie/backend_server/persona/prompt_template/run_gpt_prompt.py

```diff
@@ -2711,8 +2782,16 @@ def get_fail_safe():
   prompt = generate_prompt(prompt_input, prompt_template)
   print (prompt)
   fail_safe = get_fail_safe() 
-  output = ChatGPT_safe_generate_response_OLD(prompt, 3, fail_safe,
-                        __chat_func_validate, __chat_func_clean_up, verbose)
+  #output = ChatGPT_safe_generate_response_OLD(prompt, 3, fail_safe,__chat_func_validate, __chat_func_clean_up, verbose)
+  output = generate_structured_response(
+        prompt,
+        gpt_param,
+        SafetyScore,
+        3,
+        fail_safe,
+        __chat_func_validate,
+        __chat_func_clean_up
+    )
```

Comment:
Let's also wait on this one since it was using `ChatGPT_safe_generate_response_OLD`